components/single-form.jsx

```diff
@@ -136,13 +136,8 @@ module.exports = React.createClass({
     this.stripeCheckout(validate, submit, this.props.billingAddress);
   },
   renderPrivacyPolicy: function() {
-    if (this.props.privacyPolicyTest) {
-      return (
-        <p className="full"><FormattedHTMLMessage message={this.props.privacyPolicyTest}/></p>
-      );
-    }
     return (
-      <PrivacyPolicyCheckbox name="privacyPolicy"/>
```

Comment:
I removed it from this file, which I missed.

If you meant delete the actual component at it's source, I did not, because signup still uses it.